security/remember_me.rst

```diff
@@ -132,6 +132,15 @@ The ``remember_me`` firewall defines the following configuration options:
     Defines the service id of a token provider to use. If you want to store tokens
     in the database, see :ref:`remember-me-token-in-database`.
 
+``service`` (default value: ``null``)
+    Defines the ID of the service used to handle the Remember Me feature. It's
+    mostly useful in complex applications that need to override the default
```

Comment:
I would remove `complex`, even the whole sentence maybe.

Reply:
I made some adjustments: bbfdec196ec90900a56f9f4bab0138489f984713